nets.ts

```diff
@@ -0,0 +1,16 @@
+
+import { bins, net } from "capi/nets"
+
+const bin = bins({
+  polkadot: ["polkadot-fast", "v0.9.38"],
+  polkadotParachain: ["polkadot-parachain-fast", "v0.9.380"],
```

Comment:
```suggestion
```

Reply:
Is that no longer a valid bin specification?

Reply:
Ah, good catch T6. It is valid, but it's unused.

Reply:
*(deprecated)*